icon_validator/rules/support_validator.py

```diff
@@ -0,0 +1,39 @@
+from .validator import KomandPluginValidator
+
+
+class SupportValidator(KomandPluginValidator):
+
+    @staticmethod
+    def validate_support(support):
+        lsupport = support.lower()
+        if lsupport == 'komand':
+            raise Exception("Support 'komand' not allowed. It's likely you meant 'rapid7'.")
+        if support.endswith('.'):
+            raise Exception('Support ends with period when it should not.')
```

Comment:
```suggestion
            raise Exception("Support ends with period when it should not.")
```
Make sure to keep quotation style consistent within the project

Reply:
the top line use " because the string contains ' that way you don't have to escape.